packages/editor/src/bindings/post-meta.js

```diff
@@ -32,6 +32,11 @@ export default {
 			} );
 	},
 	canUserEditValue( { select, context, args } ) {
+		// Lock editing in query loop.
```

Comment:
Should it happen for all sources instead l?

Reply:
I am not sure if it should happen in all the sources. I am thinking about pattern overrides, for example. Maybe you want to include a synced pattern there. To be honest, I don't have a strong opinion about what is the desired behavior. I just considered at that time that it was safer to lock it for post-meta until we figure out other use cases.

By the way, should we move this conversation to the original pull request where this was disabled: [link](https://github.com/WordPress/gutenberg/pull/63237)? I don't know why after the rebase this change was included in the diff, but it doesn't belong to this pull request.